src/tests/stub_IpcIoFile.cc

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 1996-2020 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "squid.h"
+
+#if USE_DISKIO
```

Comment:
Can we use the more specific `HAVE_DISKIO_MODULE_IPCIO` here?

Reply:
Done (e382f47).

Reply:
Done (af443a7).